### DISCUSSION TITLE
Support without docker

### DISCUSSION THREAD

**Discussion by dbilgin** [serverpod/serverpod#101] [Q&A] [answered]
I feel like especially for simple projects having to set the project up with docker is a bit of an overkill. Is there a reason why the initial project can't just be created without docker and run like that?

**Comment by vlidholt** [comment, marked as answer]
Hi! Serverpod requires Postgres and Redis to run. You can configure those manually and just edit the config file of your newly created project. However, it's probably more work to setup Postegres and Redis than to just install Docker Desktop.

**Comment by dbilgin** [comment]
Yes, I agree of course it is going to be more work to prepare postgres and redis separately and honestly I really like this project. I may be alone in thinking this, but many times when I am starting up a project, I don't need redis, even though I agree that it's a great tool, sometimes what I just want to do is create a small project just to test things out and maybe put it on my server to see how it works. I think in this case the whole docker thing becomes an overkill.
I think in a lot of instances it might be nice to have a simplified version of the initial project like node express or python's flask. I mean in general almost no framework forces docker and databases etc, right?
